actions/send.email.yaml

```diff
@@ -0,0 +1,57 @@
+---
+description: Send Email
+enabled: true
+entry_point: src/send_email.py
+name: send.email
+parameters:
+    send_as_html:
+        description: "If True Send Email as HTML, else as plain text"
+        required: true
+        type: boolean
+        default: true
+    admin_override:
+        description: "Override sending emails to instead send to one test email address"
+        required: true
+        type: boolean
+        default: false
+    admin_override_email:

```

Comment:
I think this should be dropped
If we have "test-run" (or equiv) enabled we should use the email address (we enter the admin one instead) and send a single or random subset (rather than all) of the emails to this address